packages/todos-lib/client.js

```diff
@@ -0,0 +1,11 @@
+/* global TodosLib:true */
+
+// I'm not sure exactly where this should go
+TodosLib = {
+  showError(error) {
+    if (error) {
+      // Use the global __ so that we can get symbols from other packages
```

Comment:
What's the difference between global and local here? Is there some other `__` that we could be using? Not too familiar with the package.